addons/mail/static/src/core/common/thread_model.js

```diff
@@ -983,6 +983,17 @@ export class Thread extends Record {
         }
     }
 
+    replaceMessages(tmpMsg, message) {
```

Comment:
if message is always set and tmpMsg not, I would inverse the order of the params